dotnet-efcore/RecordsAsModelClasses/RecordsAsModelClasses/Controllers/v1/CarsController.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using RecordsAsModelClasses.Core.Context;
+using RecordsAsModelClasses.Core.DTOs;
+using RecordsAsModelClasses.Core.Entities.Records;
+
+namespace RecordsAsModelClasses.Controllers.v1;
+
+[Route("api/v1/cars")]
+[ApiController]
+public class CarsController : ControllerBase
+{
+    private readonly CarDbContext _context;
+
+    public CarsController(CarDbContext context)
+    {
+        _context = context;
+    }
+
+    [HttpPost]
+    public async Task<IActionResult> CreateCarAsync()
+    {
+        var car = new RecordCar(1, "Toyota", "Crown", 1952);
+
+        _context.RecordCars.Add(car);
+        await _context.SaveChangesAsync();
+
+        return Created("api/v1/cars", car);
```

Comment:
We should create a GET endpoint, that take the id of the car and returns it (Let's call it GetCar(int id)) . and this code should then be : 
```suggestion
        return CreatedAtAction(nameof(GetCar), new {id = id}, car);
```

Reply:
We should do this for both version of controller

Reply:
Yes, for both Controllers, I've added GET endpoints. Please take a look

Reply:
Please pay attention to where I have suggested the CreatedAtAction, it's in the create method not the Get method.

Reply:
And please update the article code as well for this and the other controller. Thanks!